tools/autograd/derivatives.yaml

```diff
@@ -1669,6 +1670,7 @@
 
 - name: constant_pad_nd(Tensor self, int[] pad, Scalar value=0) -> Tensor
   self: constant_pad_nd_backward(grad, pad)
+  result: auto_linear
```

Comment:
There is no test for this one?

Reply:
Added now. This was also used be conv2d, so I didn't look for the OpInfo at first.